pyworkflow/gui/project/viewprotocols.py

```diff
@@ -2211,7 +2211,11 @@ def _analyzeResults(self, prot):
             else:
                 firstViewer.visualize(prot)
         else:
+            outputList = []
```

Comment:
Why do we need to use this intermediate list here?

Reply:
We have a case now where a viewer (CTFTomo viewer) serves as a "selector" to choose, after viewing the results, those CTF that need to be redone. The viewer, generates a subset with the "bad" ones as an output and, the output generated was shown afterwards.

Basically, to prevent the Scipion to open outputs generated by viewers right afterwards they were created

Reply:
Ok, got it